experimental/PropertyDDS/packages/property-properties/package.json

```diff
@@ -94,6 +94,15 @@
 		"sinon": "^17.0.1",
 		"typescript": "~5.4.5"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"tsc": [
+				"...",
+				"@fluid-experimental/property-common#build:esnext",
```

Comment:
The other PR didn't do anything for `property-common`. Why would it be needed here and no similar change in other PR?

Reply:
I got my wires crossed. This can be removed.

Reply:
```suggestion
```